repos/fdbt-site/src/utils/apiUtils/validator.ts

```diff
@@ -42,6 +42,12 @@ export const invalidCharactersArePresent = (value: string): boolean => {
 
     return regularExpression.test(value);
 };
+export const invalidUrlInput = (value: string): boolean => {
```

Comment:
Could we add some unit tests for this function with some different inputs, to show that it works for what we expect (www., https:// etc) and rejects nonUrls.